open-company/open-company-web
# Quick post

Card: https://trello.com/c/MnoehHRG

Review with:
- [Storage](https://github.com/open-company/open-company-storage/pull/213)

To test:
- add a post via quick post
- add a post via quick post with a GIF
- add a post via quick post with a video
- add a post via quick post with an image
- add a post via quick post with attachments
- start quick post
- switch to fullscreen
- dismiss using the X
- start quick post again
- [x] quick post is cleaned up?
- [x] can you add a video now?
- refresh
- start a quick post and add some text
- refresh the page
- [x] are you sent to fullscreen edit with the text you wrote before?
- refresh
- start quick post
- add some text
- navigate to a section
- [x] quick post is now reset and collapsed?
- [x] expand and make sure the title and body are empty
- add some text again
- expand a post
- [x] no quick post in expanded post view?
- go back to section/AP
- [x] quick post is collapsed?
- [x] expand it and make the title and body are empty
- add some text
- post it
- [x] do any other test you can think of on quick post interactions
